### ISSUE TITLE
bootcmd -- not working for me

### ISSUE THREAD

**Issue by PaulCharlton** [rancher/os#1968]
**RancherOS Version: (ros os version)**

**Where are you running RancherOS? (docker-machine, AWS, GCE, baremetal, etc.)**

**Comment by PaulCharlton** [comment]
I see why bootcmd is not running for me.   bootstrap is occurring before cloudinit and before switchroot, which means that the only cloud init available to bootstrap is the one provided to the docker-machine driver (which in my case is virtualbox, which does not provide one)  re: `init/init.go`.   The configs on the RANCHER_STATE persistent drive are not yet mounted or scanned.

This is not resolved (yet)

**Comment by SvenDowideit** [comment]
yeah, I'm thinking about removing bootcmd - I'm struggling to find a place that it can actually work, or a functionality that could be used.
